tools/maf_clean_up.py

```diff
@@ -0,0 +1,177 @@
+input_maf = open ("data_mutations_extended.txt", "r")
+header_line = "";
+
+# search for header row
+for line in input_maf:
+	values = line.split("\t");
+	if values[0].upper() == "HUGO_SYMBOL":
+		header_line = line;
```

Comment:
just in general -- we don't need the semi-colon at the end of these lines. Might be better to standardize and take it out.

Reply:
fixed